src/gpo/assets/PowerToys.admx

```diff
@@ -59,6 +59,16 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
+    <policy name="ConfigureEnabledUtilityCmdNotFound" class="Both" displayName="$(string.ConfigureEnabledUtilityCmdNotFound)" explainText="$(string.ConfigureEnabledUtilityDescription)" key="Software\Policies\PowerToys" valueName="ConfigureEnabledUtilityCmdNotFound">
+      <parentCategory ref="PowerToys" />
+      <supportedOn ref="SUPPORTED_POWERTOYS_0_71_0" />
```

Comment:
Please update this shortly before merging the PR.

Reply:
I have this with some additional changes locally. Need to do some more tests before pushing the changes